### DISCUSSION TITLE
auth/core - In Vercel environment a single set-cookie with two cookies

### DISCUSSION THREAD

**Discussion by canastro** [nextauthjs/next-auth#6804] [Help] [answered]
My google sign-in and signout (if I sign-in with email) are currently failing in Vercel enviroments, and my best guess is due to the response only having a single set-cookie with two cookies.

The response to POST request to: `https://arro-main-1h6ku3ok0-arro.vercel.app/auth/signout` returns two cookies in a single `set-cookie` entry and Signout simply doesnt clear the session cookie.
![image](https://user-images.githubusercontent.com/1267623/221168006-bc3aadfa-9417-4013-9e39-88e4353a7fb3.png)

POST request to ` https://arro-main-1h6ku3ok0-arro.vercel.app/auth/signin/google?` also sets two cookies in a single set-cookie, and then I get a "Invalid code verifier." in my logs. 
![image](https://user-images.githubusercontent.com/1267623/221167378-64bd7fd8-b2e1-4021-9301-0927a157a0c4.png)

If I run locally (with exactly the some code) everything works as expected with two set-cookie entries (for both endpoints):
![Screenshot 2023-02-23 at 18 28 39](https://user-images.githubusercontent.com/1267623/220998524-9f86834f-2f25-4ebf-a520-d88c7c0a8096.png)

I'm using @auth/core@0.4.0 and @auth/sveltekit@0.2.1

**Comment by atridadl** [comment]
I didn't nail down what was causing my issues! I wonder when this was introduced... I cannot seem to find a combination of those packages that works...

My issue: https://github.com/nextauthjs/next-auth/issues/6791

**Comment by atridadl** [comment]
Just to add to this, it appears to be an issue as far back as 0.2.5, so I am hesitant to say that it was due to that package specifically. I don't recall encountering this at all until a yesterday-ish.

**Comment by atridadl** [comment, marked as answer]
Looks to be resolved when updating sveltekit

**Reply by canastro** [reply]
Awesome, will give it a try

**Reply by atridadl** [reply]
How did it go?

**Reply by canastro** [reply]
Working perfectly 👌🏽
